src/main/java/redis/clients/jedis/resps/KeyedListElement.java

```diff
@@ -2,13 +2,16 @@
 
 import redis.clients.jedis.util.SafeEncoder;
 
+import java.awt.*;
+
 /**
  * This class is used to represent a List element when it is returned with respective key name.
  */
 public class KeyedListElement {
 
   private final String key;
   private final String element;
+  private final List elementList;
```

Comment:
@sazzad16 added a new list value type here.Please suggest can we proceed this way.

Reply:
@Razi007 I don't like how it has been handled later, like `this.elementList = new List();` and `this.element = "";`.